src/DomainModel/DomainModelAwareTrait.php

```diff
@@ -54,10 +54,10 @@ public function loadService($model, array $constructorArgs = [], $assignProperty
             return $domainModel;
         }
 
-        list(, $name) = pluginSplit($model);
+        [, $name] = pluginSplit($model);
 
         if (isset($this->{$name})) {
-            trigger_error(__CLASS__ . '::$%s is already in use.', E_USER_WARNING);
+            trigger_error(self::class . '::$%s is already in use.', E_USER_WARNING);
```

Comment:
```suggestion
            trigger_error(static::class . '::$%s is already in use.', E_USER_WARNING);
```

I think `static` is better,
it refers to last class, not where this trait used

Reply:
This was done by sniffers.

Reply:
also forgot to use the `sprintf()`  here

Reply:
I checked again replace of `__CLASS__` exactly  is `self::class`, but probably `static::class` is true here, not tested locally